.github/ISSUE_TEMPLATE/01-react-components-bug-report.yml

```diff
@@ -0,0 +1,213 @@
+    attributes:
+      label: Component
+      description: |
+        Which Component/Utility is affected
+      options:
+        - Accordion
+        - Avatar
+        - AvatarGroup
+        - Breadcrumb
+        - Checkbox
+        - Combobox
+        - DataGrid
+        - Dialog
+        - Divider
+        - Drawer
+        - Dropdown
+        - Field
+        - FluentProvider
+        - Image
+        - InfoLabel
+        - Input
+        - Label
+        - Link
+        - MessageBar
+        - Overflow
+        - Persona
+        - Popover
+        - ProgressBar
+        - RadioGroup
+        - Rating
+        - RatingDisplay
+        - SearchBox
+        - Select
+        - Skeleton
+        - Slider
+        - SpinButton
+        - Spinner
+        - SwatchPicker
+        - Switch
+        - Table
+        - TabList
+        - TagPicker
+        - TeachingPopover
+        - Text
+        - Textarea
+        - Toast
+        - Toolbar
+        - Tooltip
+        - Tree
+        - Calendar (Compat)
+        - DatePicker (Compat)
+        - TimePicker (Compat)
+        - Carousel (Preview)
+        - List (Preview)
+        - Nav (Preview)
+        - Virtualizer (Preview)
+        - Motion
+        - Icons
+        - Theme/Tokens
+        - Migration Shims v0
+        - Migration Shims v8
+        - Utilities (utilities we provide besides Components, e.g. apis from react-utilities)
+        - Other...
+    validations:
+      required: true
+
+  - type: input
+    id: pkg-version
+    attributes:
+      label: Package version
+      description: |
+        Which version of react-components are you using?
+      placeholder: e.g. 9.42.0
+    validations:
+      required: true
+
+  - type: input
+    id: react-version
+    attributes:
+      label: React version
+      description: |
+        Which version of React are you using?
+      placeholder: e.g. 17.0.2
+    validations:
+      required: true
+
+  - type: textarea
+    id: env-info
+    attributes:
+      label: Environment
+      description: |
+        Output of `npx envinfo@latest --system --browsers --npmPackages '{@fluentui/*,react,react-dom,@types/react{,-dom}}'`
+        NOTE 💡: This will be automatically formatted as a code block, so no need for backticks.
+      render: shell
+    validations:
+      required: true
+
+  - type: textarea
+    id: current-behavior
+    attributes:
+      label: Current Behavior
+      description: |
+        What is the current behavior that you are experiencing?
+    validations:
+      required: true
+
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected Behavior
+      description: |
+        What is the behavior that you expect to happen?
+    validations:
+      required: true
+
+  - type: input
+    id: reproduction-link
+    attributes:
+      label: Reproduction
+      description: |
+        A link to a Stackblitz reproduction which demonstrates the bug - https://stackblitz.com/
+
```

Comment:
```suggestion
> 💡 You can export to Stackblitz any example from docs (https://react.fluentui.dev/) with "Open In StackBlitz" button and update it to reproduce an issue.
```

Let's mention this as creating an example from scratch requires a _bit_ of boilerplate.